myfitnesspal/entry.py

```diff
@@ -1,11 +1,22 @@
 from myfitnesspal.base import MFPBase
+import re
```

Comment:
Mega nit, but PEP-8 asks that our standard library imports be in a separate block from our local imports; I think the imports section might match that if it were instead something like:

```python
import re

from myfitnesspal.base import MFPBase
```

I should really enable the linter for this library so the robot can make these comments instead of me :-); sorry for the tedium.